### PR TITLE
Create an animation to run during the case uploading

### PR DESCRIPTION
**Pull request by tatkaef**

### CODE REVIEW

**Review by ndossett — APPROVED** [review]
This is something the stakeholder asked for specifically - nice to have it checked off. In the future, keep in mind we are trying to use all Ant Design for new components when possible.

**Review by JusticeMatthew — APPROVED** [review]
Thank you for taking care of this loader!

**Review by ruizaj13 — APPROVED** [review]
Fulfills a stakeholder request and greatly improves user experience, great work!

File: src/components/pages/Upload/UploadCase.js [hunk 1]
```diff
@@ -2,8 +2,9 @@ import React, { useState } from 'react';
 import axios from 'axios';
 import { useForm } from 'react-hook-form';
 import Button from '@material-ui/core/Button';
-import { makeStyles } from '@material-ui/core/styles';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
 import UploadCaseForm from './UploadCaseForm';
```
Comment (crystal-csete) [inline src/components/pages/Upload/UploadCase.js:6]: This was needed, thank you! Moving forward use ant design when possible.

**Review by crystal-csete — APPROVED** [review]

**Pull request merged**